src/status_im/ui/screens/signing/views.cljs

```diff
@@ -101,7 +101,7 @@
       [react/nested-text
        {:style           {:color colors/gray}
         :ellipsize-mode  :middle
-        :number-of-lines 1} (i18n/label :t/to) " "
+        :number-of-lines 1} (i18n/label :t/To) " "
```

Comment:
@vkjr, I think we shouldn't use uppercased keywords in status-mobile . If we need to differentiate between keywords that already exist, we can just give them a different name. The translation key doesn't need to match the text. Mixed case keywords are too alien in Clojure, I've never seen this.